src/test/java/model/process/setOperations/DifferenceTest.java

```diff
@@ -0,0 +1,261 @@
+package model.process.setOperations;
+
+import model.data.DataModel;
+import model.data.DataRow;
+import model.data.DataTable;
+import model.data.DataTableBuilder;
+import model.data.value.FloatValue;
+import model.data.value.StringValue;
+import model.language.Identifier;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Created by jens on 6/4/15.
+ */
+public class DifferenceTest {
+
+	@Test(expected = Exception.class)
```

Comment:
Oh, inderdaad. Je krijgt hier volgens mij een IllegalArgument exception, dus dat is zo gefixt.